src/ts/helpers/parse.ts

```diff
@@ -128,7 +128,14 @@ const htmlChars = new RegExp(Object.keys(htmlCharMap).join("|"), "g");
  * Escapes unsafe characters is a string to render safely in HTML
  * @param  {string} unsafe - string to be rendered in HTML
  */
-export function escapeHtml(unsafe: string) {
+export function escapeHtml(unsafe: string | number | boolean = "") {
+  if (typeof unsafe !== "string") {
```

Comment:
You're right - technically yes, for our current use it's not needed (though they get stringified anyway).
My motivation for doing this is to make it type-safe to return strings no matter what (e.g. to call another `replace` on it).